Source/Scene/Scene.js

```diff
@@ -2696,6 +2723,68 @@ define([
         return object;
     };
 
+    function renderTranslucentDepthForPick(scene, drawingBufferPosition) {
+        // PERFORMANCE_IDEA: render translucent only and merge with the previous frame
+        var context = scene._context;
+        var us = context.uniformState;
+        var frameState = scene._frameState;
+
+        // Update with previous frame's number and time, assuming that render is called before picking.
+        updateFrameState(scene, frameState.frameNumber, frameState.time);
+        frameState.cullingVolume = getPickCullingVolume(scene, drawingBufferPosition, 1, 1);
+        frameState.passes.pick = true;
+        frameState.passes.depth = true;
+
+        us.update(frameState);
```

Comment:
Offline we discussed using the commands from the previous frame.  Why didn't you take that approach?  Because some primitives will want to generate different commands here?  Which precisely since pick ids won't matter.

If we stick with this approach, let's add a `PERFORMANCE_IDEA` about reusing the commands.

I've been thinking about this in three levels of reuse/efficiency:
* The most efficient is what was previously done; reuse the previous frame's depth buffer.  This is not possible unless the render pass writes an extra depth buffer with translucent primitives.
* Reuse the previous frame's commands.  From a GL perspective, this is no more efficient, but it saves CPU overhead in Cesium.
* Reuse nothing from the previous frame, and explicitly `update`/`render`.